server/main.cpp

```diff
@@ -104,6 +123,10 @@ int main() {
 						{
 							std::lock_guard lg(server.client_list_mutex);
 							for (std::shared_ptr<tec::networking::ClientConnection> client : server.GetClients()) {
+								if (!client->ReadyToReceive()) {
```

Comment:
Prevents most getting flooded issues as we must wait for the client to ACK receiving the world.